test/Parse/explicit_lifetime_dependence_specifiers.swift

```diff
@@ -0,0 +1,118 @@
+// RUN: %target-typecheck-verify-swift -disable-availability-checking -enable-experimental-feature NonescapableTypes -disable-experimental-parser-round-trip   -enable-experimental-feature NoncopyableGenerics -enable-builtin-module
+// REQUIRES: asserts
+
+import Builtin
+
+struct BufferView : ~Escapable {
+  let ptr: UnsafeRawBufferPointer
+  init(_ ptr: UnsafeRawBufferPointer) {
+    self.ptr = ptr
+  }
+}
+
+struct MutableBufferView : ~Escapable, ~Copyable {
+  let ptr: UnsafeMutableRawBufferPointer
+  init(_ ptr: UnsafeMutableRawBufferPointer) {
+    self.ptr = ptr
+  }
+}
+
+func testBasic() {
+  let capacity = 4
+  let a = Array(0..<capacity)
+  a.withUnsafeBytes {
+    let view = BufferView($0)
+    let derivedView = derive(view)
+    let newView = consumeAndCreate(derivedView)
+    use(newView)    
+  }
+}
+
+func derive(_ x: borrowing BufferView) -> _borrow(x) BufferView {
+  return BufferView(x.ptr)
+}
+
+func consumeAndCreate(_ x: consuming BufferView) -> _copy(x) BufferView {
+  return BufferView(x.ptr)
+}
+
+func use(_ x: borrowing BufferView) {}
+
+func deriveMultiView1(_ x: borrowing BufferView, _ y: borrowing BufferView) -> _borrow(x, y) BufferView {
+  if (Int.random(in: 1..<100) % 2 == 0) {
+    return BufferView(x.ptr)
+  }
+  return BufferView(y.ptr)
+}
+
+func deriveMultiView2(_ x: borrowing BufferView, _ y: borrowing BufferView) -> _borrow(x) _borrow(y) BufferView {
```

Comment:
Is it intentional that it’s allowed to specify `_borrow` twice? I think we should error here and only allow `_borrow(x, y)`.

Reply:
I had the same opinion. @atrick suggested not to diagnose this previously. 

If it's going to be a problematic grammar, we can ban it.

cc @atrick